configs/detection/vitdet/vitdet_schedule_100e.py

```diff
@@ -1,26 +1,29 @@
 _base_ = 'configs/base.py'
```

Comment:
What does the name of this file mean vitdet_schedule_100e.py？

Reply:
vitdet training plan，total 100 epoch